WcaOnRails/app/views/admin/avatars/index.html.erb

```diff
@@ -4,6 +4,18 @@
   <%= render layout: "nav" do %>
     <h1><%= yield(:title) %></h1>
 
+    <div class="well">
+      <h3>Guidelines</h3>
+      <ul>
+        <li>All avatars should be pictures of real persons.</li>
```

Comment:
This is intentional. Basically the sentences differ a bit because this is meant for admin to read, whereas on the user profile we use words like `you` instead of `the person`. Also I think that we agreed that admin pages don't have to be localized as admins know english well =)